src/Compilers/CSharp/Test/Syntax/Parsing/PartialEventsAndConstructorsParsingTests.cs

```diff
@@ -0,0 +1,1061 @@
+            partial event Action E = null, F = null;
+            """);
+
+        N(SyntaxKind.EventFieldDeclaration);
+        {
+            N(SyntaxKind.PartialKeyword);
+            N(SyntaxKind.EventKeyword);
+            N(SyntaxKind.VariableDeclaration);
+            {
+                N(SyntaxKind.IdentifierName);
+                {
+                    N(SyntaxKind.IdentifierToken, "Action");
+                }
+                N(SyntaxKind.VariableDeclarator);
+                {
+                    N(SyntaxKind.IdentifierToken, "E");
+                    N(SyntaxKind.EqualsValueClause);
+                    {
+                        N(SyntaxKind.EqualsToken);
+                        N(SyntaxKind.NullLiteralExpression);
+                        {
+                            N(SyntaxKind.NullKeyword);
+                        }
+                    }
+                }
+                N(SyntaxKind.CommaToken);
+                N(SyntaxKind.VariableDeclarator);
+                {
+                    N(SyntaxKind.IdentifierToken, "F");
+                    N(SyntaxKind.EqualsValueClause);
+                    {
+                        N(SyntaxKind.EqualsToken);
+                        N(SyntaxKind.NullLiteralExpression);
+                        {
+                            N(SyntaxKind.NullKeyword);
+                        }
+                    }
+                }
+            }
+            N(SyntaxKind.SemicolonToken);
+        }
+        EOF();
+    }
+
+    [Fact]
+    public void Event_Definition_PartialAfterEvent()
+    {
+        UsingDeclaration("""
+            event partial Action E;
+            """,
+            null,
+            // (1,7): error CS1031: Type expected
+            // event partial Action E;
+            Diagnostic(ErrorCode.ERR_TypeExpected, "partial").WithLocation(1, 7),
+            // (1,7): error CS1525: Invalid expression term 'partial'
+            // event partial Action E;
+            Diagnostic(ErrorCode.ERR_InvalidExprTerm, "partial").WithArguments("partial").WithLocation(1, 7),
+            // (1,7): error CS1003: Syntax error, ',' expected
+            // event partial Action E;
+            Diagnostic(ErrorCode.ERR_SyntaxError, "partial").WithArguments(",").WithLocation(1, 7));
+
+        N(SyntaxKind.EventFieldDeclaration);
+        {
+            N(SyntaxKind.EventKeyword);
+            M(SyntaxKind.VariableDeclaration);
+            {
+                M(SyntaxKind.IdentifierName);
+                {
+                    M(SyntaxKind.IdentifierToken);
+                }
+                M(SyntaxKind.VariableDeclarator);
+                {
+                    M(SyntaxKind.IdentifierToken);
+                }
+            }
+            N(SyntaxKind.SemicolonToken);
+        }
+        EOF();
+    }
+
+    [Fact]
+    public void Event_Definition_PartialAfterType()
+    {
+        UsingDeclaration("""
+            event Action partial E;
+            """,
+            null,
```

Comment:
Please label the argument (same comment for below usages of `null` literal arguments)

Reply:
These should be gone now since I pass the options explicitly to test different lang versions.